content/blog/flux-cncf-graduation/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Flux Graduates within CNCF! 🥳
+date: "2022-11-30"
+description: "Flux Graduates within CNCF!"
+author: Scott Rigby
+thumbnail: images/flux-cncf-graduation.png
+---
+
+Today we celebrate [Flux's graduation within CNCF](#ADD-CNCF-LINK)! A momentous occasion not only for the Flux project, but for the GitOps Working Group and OpenGitOps project.
+Flux (and its subproject, Flagger) has been providing GitOps and Progressive Delivery capabilities to a growing community of users, enterprise companies, and providers such as Weaveworks, AWS, Microsoft, D2iQ, VMware, and Red Hat.
+So it's no surprise that the project met the scrutiny of the Cloud Native Computing Foundation's Technical Oversight Committee and its third-party security audits.
+Flux has come a long way since it's beginnings as the project that [led to the term, GitOps](https://www.weave.works/blog/the-history-of-gitops).
```

Comment:
```suggestion
Flux has come a long way since its beginnings as the project that [led to the term, GitOps](https://www.weave.works/blog/the-history-of-gitops).
```